Simcoe/mParticle/mParticle.swift

```diff
@@ -9,12 +9,12 @@
 import mParticle_Apple_SDK
 
 /// Simcoe Analytics handler for the MParticle iOS SDK.
-public class mParticle {
+open class mParticle {
 
-    private static let unknownErrorMessage = "An unknown error occurred."
+    fileprivate static let unknownErrorMessage = "An unknown error occurred."
```

Comment:
Yup! Because we are using that variable in an extension -- the access modifier has to be fileprivate